docs/overview.md

```diff
@@ -115,3 +115,46 @@ Expand, Skip, Top, OrderBy, and Filter are also supported via the client library
 ## Collections
 
 Please see [collections](/docs/collections.md) for details on collections and paging.
+
+## Use the .Net Microsoft Graph client library when the client doesn't support a feature
```

Comment:
I think this is a bit confusing if someone isn't familiar with the library :) Use the library when the library doesn't support x.  Maybe rephrase to something like "sending generic requests with the client library"?